README.adoc

```diff
@@ -53,7 +53,7 @@ The `inventory` service is built in a similar way.
 === What is CDI?
 
 Contexts and Dependency Injection (CDI) defines a rich set of complementary services that improve the application structure.
-The most fundamental services that are provided by CDI are contexts that bind the lifecycle of stateful components to well-defined contexts,
+The most fundamental services that are provided by CDI are scopes that bind the lifecycle of stateful components to well-defined contexts,
```

Comment:
Thank you for the PR. 

Actually the contexts here is referring to the term in CDI. `scopes` are well-defined but extensible lifecycle contexts. They're different meaning. The original is accurate.

Reply:
By the way, the original sentence is inline with CDI spec.